src/views/mainLayout.js

```diff
@@ -42,6 +42,7 @@ const mainLayout = data => `
     <link rel='apple-touch-icon' href='/images/apple-touch-icon.webp' sizes='180x180'>
 
     <script src='/js/index.js' type='module'></script>
+    ${data.skipPartialModule ? '' : `<script src='/js/partials/${data.partial.name}.js' type='module'></script>`}
```

Comment:
@flozia do you mean accommodating no-script users?  I think it could be helpful for promotion pages (Landing for ex), but if script is disabled for landing, the newly added form simply doesn't work – the entire rest of the page should still function as a promo page sharing content.  Perhaps we could audit the site with script turned off to see what the impact is – at a certain point (past the landing) our site is just not usable without script...